_posts/en/newsletters/2024-01-17-newsletter.md

```diff
@@ -0,0 +1,305 @@
+  output amount, which is reasonable given that the policy rules for
+  them require that they be accompanied by a child transaction spending
+  the anchor output.  However, in the current LN protocol, when a party
+  wants to create an [uneconomic][topic uneconomical outputs] HTLC, the
+  payment amount is instead used to overpay the commitment transaction's
+  onchain fees; this is called a _trimmed HTLC_.  If HTLC trimming is
+  done in a commitment transaction using ephemeral anchors, it could be
+  profitable for a miner to confirm the commitment transaction without a
+  child transaction that spends the ephemeral anchor output.  Once a
+  commitment transaction is confirmed, there's no incentive for anyone
+  to spend a zero-amount ephemeral anchor output, meaning it will
+  consume space in full node UTXO sets forever, an undesirable outcome.
+
+  A proposed alternative is to put the trimmed HTLC amounts into the
+  value of the ephemeral anchor outputs.  That way they incentivize
+  confirming both the commitment transaction and a spend of the
+  ephemeral anchor output.  In his post, Sanders analyzes this
+  possibility and finds that this can create several security problems.
+  These can be solved by miners analyzing transactions, determining when
+  it would be more profitable for them to spend an ephemeral anchor
+  output themselves, and creating the necessary transaction.  This is a
+  type of [miner extractable value][news201 mev] (MEV).  Several
+  alternative solutions were also proposed:
+
+  - *Only relaying transactions that are fully miner incentive compatible:*
+    if anyone attempts to spend an ephemeral anchor in a way that
+    doesn't maximize revenue for a miner, that transaction would not be
+    relayed by Bitcoin Core.
+
+  - *Burn trimmed value:* instead of turning the amount of trimmed HTLCs
+    into fees, the amount would instead be spent to an `OP_RETURN`
+    output, destroying those satoshis by making them permanently
+    unspendable.  This would only happen if a commitment transaction
+    containing a trimmed HTLC was put onchain; normally trimmed HTLCs
+    are resolved offchain and their value successfully transferred from
+    one party to the other.
+
+  - *Ensure MEV transactions propagate easily:* instead of having miners
+    run special code that maximizes their value, ensure that
+    transactions maximizing their value propagate easily through the
+    network.  That way anyone can run the MEV code and relay the results
+    to miners in a way that ensures all miners and relay nodes can
+    obtain the same set of transactions.
+
+  No clear conclusion seemed to have been reached at the time of writing.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.0.119][] in a new release of this library for building
```

Comment:
```suggestion
- [LDK 0.0.119][] in a new release of this library for building
```

Reply:
Thank you @vostrnad!  I just realized I missed the correction from LND to LDK.